server/remote/msgraph/get_event_delta.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package msgraph
+
+import (
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+)
+
+type getEventDeltaResponse struct {
+	NextLink  string          `json:"@odata.nextLink"`
+	DeltaLink string          `json:"@odata.deltaLink"`
+	Value     []*remote.Event `json:"value"`
+}
+
+func (c *client) GetEventDeltaFromDateRange(remoteUserID string, start, end *remote.DateTime) (events []*remote.Event, deltaLink string, err error) {
+	q := url.Values{}
+	q.Add("StartDateTime", start.Time().Format(time.RFC3339))
+	q.Add("EndDateTime", end.Time().Format(time.RFC3339))
+	params := "?" + q.Encode()
+
+	u := c.rbuilder.Me().CalendarView().URL() + "/delta" + params
+	var out getEventDeltaResponse
+
+	_, err = c.CallJSON(http.MethodGet, u, nil, &out)
```

Comment:
I must be missing some details here, can you call the ...FromURL here?

Reply:
Brilliant, fits like a glove.